### ISSUE TITLE
Message bar icons don't meet high contrast ratios

### ISSUE THREAD

**Issue by nipunja** [OfficeDev/office-ui-fabric-react#3262]
Issue found using keros for web chrome accessbility extension. 

Message bar icons don't meet high contrast ratios.

Fix any of the following: Element has insufficient color contrast of 4.12 (foreground color: #767676, background color: #fff4ce, font size: 12.0pt, font weight: normal)

**Comment by betrue-final-final** [comment]
Yes, that should be neutralSecondary (#666666)
The specific issue is with the grey bar.

![image](https://user-images.githubusercontent.com/20363037/32186130-7ba9beda-bd5e-11e7-9ae9-df332c1ac7b8.png)

**Comment by nipunja** [comment]
Even with warning yellow bar issue is seen

**Comment by betrue-final-final** [comment]
Yes, that's correct. changing both to neutralSecondary would fix that

**Comment by betrue-final-final** [comment]
Before this closes, I'd like to clarify a couple of things:

1. This is indeed a contrast issue bug
2. This is not a **high contrast** issue. High contrast is specifically modes that can be set in an OS or browser that enables higher than standard contrast for the display.
